website/content/v1.7/talos-guides/install/cloud-platforms/aws.md

```diff
@@ -7,52 +7,102 @@ aliases:
 
 ## Creating a Cluster via the AWS CLI
 
-In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
-We assume an existing VPC, and some familiarity with AWS.
+In this guide we will create an HA Kubernetes cluster with 3 control plane nodes across 3 availability zones.
+You should have an existing AWS account and have the AWS CLI installed and configured.
 If you need more information on AWS specifics, please see the [official AWS documentation](https://docs.aws.amazon.com).
 
-### Set the needed info
-
-Change to your desired region:
+To install the dependencies for this tutorial you can use homebrew on macOS or Linux:
 
 ```bash
-REGION="us-west-2"
-aws ec2 describe-vpcs --region $REGION
+brew install siderolabs/tap/talosctl kubectl jq ipcalc curl xz
+```
+
+If you would like to create infrastructure via `terraform` or `opentofu` please see the exmaple in the [contrib repository](https://github.com/siderolabs/contrib/tree/main/examples/terraform/aws).
+
+> Note: this guide is not a production set up and steps were tested in `bash` and `zsh` shells.
+
+### Create AWS Resources
+
+We will be creating a control plane with 3 Ec2 instances spread across 3 availability zones.
+It is recommended to not use the default VPC so we will create a new one for this tutorial.
 
-VPC="(the VpcId from the above command)"
+Change to your desired region and CIDR block and create a VPC:
+
+> Make sure your subnet does not overlap with `10.244.0.0/16` or `10.96.0.0/12` the [default pod and services subnets in Kubernetes]({{% relref "../../../introduction/troubleshooting.md#conflict-on-kubernetes-and-host-subnets" %}}).
+
+```bash
+AWS_REGION="us-west-2"
+IPV4_CIDR="10.1.0.0/18"
+VPC_ID=$(aws ec2 create-vpc \
+    --cidr-block $IPV4_CIDR \
+    --output text --query 'Vpc.VpcId')
 ```
 
-### Create the Subnet
+### Create the Subnets
 
-Use a CIDR block that is present on the VPC specified above.
+Create 3 smaller CIDRs to use for each subnet in different availability zones.
 
 ```bash
-aws ec2 create-subnet \
-    --region $REGION \
-    --vpc-id $VPC \
-    --cidr-block ${CIDR_BLOCK}
+IPV4_CIDRS=( $(ipcalc -S 22 --no-decorate $IPV4_CIDR | head -n 3) )
```

Comment:
the `ipcalc` didn't work for me at all on linux (AWS cloudshell), but I just did a space separated list of IP addresses (i.e. `'10.1.0.0/24' '10.1.1.0/24' '10.1.2.0/24'` ). I didn't dig into it much though, so it might just require another package.

![image](https://github.com/user-attachments/assets/f2d43c9f-5670-4370-8b6f-11e9b47b0eea)


IDK what kind of CIDRs this generates, but you could offer something like this as a substitution:
```shell
IPV4_CIDRS=( $(printf '10.1.0.0/24\n10.1.1.0/24\n10.1.2.0/24\n') )
```

![image](https://github.com/user-attachments/assets/9092b41f-4558-48e2-a88d-89b29f652c9c)

Reply:
I think this is a bug with ipcalc packaged with brew. I'll change it to a manual step and make a note that people need to adjust it for their CIDRs